app/actors/hyrax/actors/interpret_visibility_actor.rb

```diff
@@ -111,6 +111,9 @@ def apply_visibility(env, intention)
               env.curation_concern.visibility = env.attributes[:visibility]
             end
           end
+
+          # Copy any visibility changes to the ChangeSet
+          env.change_set.read_groups = env.curation_concern.read_groups
```

Comment:
Is this supposed to be `visibility` instead of `read_groups` as the comment states?

Reply:
Not really.  The visibility is like a macro for read_groups.